### PR TITLE
fix: exempt test modules

### PR DESCRIPTION
**Pull request by ArArgon**

### CODE REVIEW

**Comment by ArArgon** [comment]
@GotenJBZ Take a look. Do these changes fix the mistake?

File: rules/move_on_aptos/lang/security/public-friend-entry.move [hunk 1]
```diff
@@ -18,5 +18,13 @@ module 0xabcdef::public_friend_entry {
     #[test]
     public(friend) entry fun signature_4() {
 
+    }
+}
+
+#[test]
```
Comment (GotenJBZ) [inline rules/move_on_aptos/lang/security/public-friend-entry.move:24]: This should be `#[test_only]`

Comment (ArArgon) [reply rules/move_on_aptos/lang/security/public-friend-entry.move]: Fixed

**Comment by ArArgon** [comment]
@GotenJBZ

**Comment by gedigi** [comment]
@zi0Black can this be merged?

**Review by zi0Black — APPROVED** [review]

**Pull request merged**